src/main/java/com/customerApi/api/services/CustomerService.java

```diff
@@ -0,0 +1,42 @@
+package com.customerApi.api.services;
+
+
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.customerApi.api.modals.Customer;
+import com.customerApi.api.repositories.CustomerRepository;
+
+
+
+
+@Service
+public class CustomerService {
+	
+	private static final Logger logger=LoggerFactory.getLogger(CustomerService.class);
+	
+	@Autowired
+	CustomerRepository customerRepo ;
+	
+	
+	public boolean addCustomerData(Customer data)
+	{		
+		boolean status = false;
+		try {
+			
+			customerRepo.save(data);
+			logger.info("Registered successfully"+data.getUsername());
```

Comment:
The proper way of printing log is below, its an example
_For example, if you want to print 1 value"_
logger.info("Customer Name {}, sign-in in", customer.getName());

_For example, if you want to print 2 values or multiple"_
logger.info("Customer First Name {}, last name {}, sign-in in", customer.getFirstName(), customer.getLastName());